### PR TITLE
Bio.AlignIO alphabet tweaks

### PR DESCRIPTION
**Pull request by peterjc**
This pull request makes small steps towards issue #2046, including fixing an oversight from #3000.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Tests/test_seq.py [hunk 1]
```diff
@@ -864,10 +864,8 @@ def test_complement_ambiguous_dna_values(self):
 
     def test_complement_ambiguous_rna_values(self):
         for ambig_char, values in sorted(ambiguous_rna_values.items()):
-            compl_values = str(
-                # Will default to DNA if niether T or U found...
-                Seq.Seq(values).complement().transcribe()
-            )
+            # Will default to DNA if niether T or U found...
```
Comment (mdehoon) [inline Tests/test_seq.py:867]: neither

Comment (peterjc) [reply Tests/test_seq.py]: Thanks

File: Tests/test_seq.py [hunk 2]
```diff
@@ -864,7 +864,7 @@ def test_complement_ambiguous_dna_values(self):
 
     def test_complement_ambiguous_rna_values(self):
         for ambig_char, values in sorted(ambiguous_rna_values.items()):
-            # Will default to DNA if niether T or U found...
+            # Will default to DNA if neither T or U found...
```
Comment (mdehoon) [inline Tests/test_seq.py:867]: nor

**Comment by peterjc** [comment]
Rebased - much of this had been done elsewhere in the meantime.

**Pull request merged**